test/unit/spec/MfaVerify_spec.js

```diff
@@ -263,12 +263,15 @@ function (Okta,
 
       if (options.u2f) {
         window.u2f = {
-          sign: function () {
-          }
+          sign: jasmine.createSpy('window-u2f-spy')
         };
-        spyOn(window.u2f, 'sign');
         if (options && options.signStub) {
-          window.u2f.sign.and.callFake(options.signStub);
+          window.u2f.sign.and.callFake(function () {
```

Comment:
Could we simplify this to use `spyOn` instead since we aren't asserting the spy gets called above?

```javascript
spyOn(window.u2f, 'sign').and.callFake(function () { ... });
```

Reply:
we do actually. when `u2f` option is true, we are expecting `u2f.sign` exists but may not test the logic after user tap the key.